src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -290,44 +408,54 @@ def __get_suite_filter(test_filter: str, suite_filter: str):
         return suite_filter_mixed
 
     @staticmethod
-    def __replace_restricted_symbols(input_string:str):
-        restricted_symbols = "!@$%^&-+`~:;\",<>?"
+    def __replace_restricted_symbols(input_string: str):
+        restricted_symbols = '!@$%^&-+`~:;",<>?'
```

Comment:
JFYI, [PEP:](https://peps.python.org/pep-0008/#string-quotes)
In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

Reply:
so the main improvement here removing backslashes and reduce the string length.